tools/convert/templates/configV2.tmpl

```diff
@@ -2,7 +2,7 @@
 ## Honeycomb Refinery Configuration ##
 ######################################
 #
-# created {{ now }} from {{ .Input }} using a template generated on 2025-02-13 at 16:52:04 UTC
+# created {{ now }} from {{ .Input }} using a template generated on 2025-03-27 at 23:12:19 UTC
```

Comment:
for files where the only change was the generation date, I usually just revert them rather than include them in the commit.